TweakPropertyGenerator/TweakPropertyGenerator/Core/LocalConfigurationParser.swift

```
@@ -9,53 +9,56 @@ extension String: Error {}
 
 class LocalConfigurationParser {
     
-    private func type(for value: Any) -> String {
+    func loadConfiguration(configurationFilePath: String) throws -> Configuration {
+        let url = URL(fileURLWithPath: configurationFilePath)
+        let data = try Data(contentsOf: url)
+        let content = try JSONSerialization.jsonObject(with: data) as! LocalConfigurationFormat
+        
+        let tweaks = try content.map { (featureKey: String, tweaks: [String: [String: Any]]) throws -> [Tweak] in
+            try tweaks.map { (variableKey: String, value: [String: Any]) throws -> Tweak in
+                try tweak(for: value, feature: featureKey, variable: variableKey)
+            }
+            .sorted {
+                $0.variable < $1.variable
+            }
+        }
+        .flatMap { $0 }
+        .sorted {
+            $0.feature < $1.feature
+        }
+        
+        let configuration = Configuration(tweaks: tweaks)
+        return configuration
+    }
+    
+    func type(for value: Any) -> String {
         switch value {
         case _ as String: return "String"
         case let numberValue as NSNumber: return numberValue.tweakType
         case _ as Bool: return "Bool"
         case _ as Double: return "Double"
-        default: return "String"
+        default: return "unkwown"
```

typo here **"unknown"** - I wonder whether we should assert/error here as it may be bad to fail silently with an incorrect value type

---

good idea, or maybe - even better - raise an exception like other methods do already